tests/dialects/test_stencil.py

```diff
@@ -205,7 +205,7 @@ def test_stencil_apply_no_args():
 def test_stencil_apply_no_results():
     # Should error if there are no results expected
     with pytest.raises(AssertionError):
-        ApplyOp.get([], Block([]), [])
+        ApplyOp.get((), Block(), ())
```

Comment:
for some reason it likes tuples more than lists

Reply:
What was the error here?